docker/Jenkinsfile-build-xgb-docker

```diff
@@ -2,6 +2,7 @@
 
 final String NODE_LABEL = 'docker && !mr-0xc8 && (!micro || micro_21)'
 final String DOCKER_STASH = 'h2o-3-xgb-docker-stash'
+final String BUILD_TAG = '1'
```

Comment:
Why we need a build tag?

Reply:
It give as an option to use different version of the docker images for xgb flavers. 

Before this change we produce images: xgb-runtime-BACKEND-RUNTIME without an option go back in xgb upgrade.

After this change we produce: xgb-runtime-BACKEND-RUNTIME-BUILD-TAG and can go to previous version.